remoulade/common.py

```diff
@@ -25,7 +25,38 @@
 from .errors import QueueJoinTimeout
 
 
-def compute_backoff(attempts, *, factor=5, jitter=True, max_backoff=2000, max_exponent=32):
+def compute_backoff(attempts, *, strategy=None, factor=5, jitter=True, max_backoff=2000, max_exponent=32):
```

Comment:
```suggestion
def compute_backoff(attempts, *, strategy: BackoffStrategy="exponential", min_backoff: int=5, max_backoff: int=2000, jitter=True) -> Tuple[int, int]:
```
I think we can remove `max_exponent`, we already have a max_retries and max_backofff

Reply:
Maybe we should rename it, but in the context of the functions that use it, max_exponent is max_retries.

Reply:
yes I saw after. Another reason to remove it from here, but you can keep it in the compute_exp function using max_retries

Reply:
But then how will we get the max_retries value, since we are not in a middleware ?

Reply:
indeed. It is just naming then, change everything to the middleware names: `min_backoff`, `max_retries`, ...